tasm-test/src/LayoutBenchmarkTest.cc

```diff
@@ -0,0 +1,198 @@
+#include "TiledVideoManager.h"
+#include "SemanticIndex.h"
+#include "VideoManager.h"
+#include "Tasm.h"
+#include <gtest/gtest.h>
+#include <chrono>
+#include <fstream>
+#include <iostream>
+#include <functional>
+
+using namespace tasm;
+
+#define VIDEO_PATHS {"/home/maureen/red_videos/red_6sec_2k.mp4", "/home/maureen/red_videos/red_900sec_2k.mp4"}
+#define LABEL_PATHS {"/home/maureen/NFLX_dataset/detections/labels/birdsincage.db", "/home/maureen/visualroad_tiling/labels_yolo/traffic-2k-001_yolo.db"}
+#define VIDEO_LABELS {"bird", "car"}
+#define VIDEO_NAMES {"birdsincage", "traffic-2k-001_yolo"}
+#define NUM_FRAMES {180, 27000}
+#define IDS {197, 8476}
+#define NUM_VIDEOS 2
+#define NUM_ITERATIONS 3
+#define FUNCTION_REPETITIONS 50 // for functions that are too fast to measure without repetition
```

Comment:
Do you think we should also include FUNCTION_REPETITIONS in the output file? (With a value of 1 for the non-repeated ones) My thought is that it's possible this could change over time, so it could be useful to know for a specific set of results what its value was.